hail/python/hail/methods/impex.py

```diff
@@ -687,18 +678,18 @@ def import_bed(path, reference_genome='default', skip_invalid_intervals=False, *
 
     if t.row.dtype == tstruct(f0=tstr, f1=tint32, f2=tint32):
         t = t.select(interval=locus_interval_expr(t['f0'],
-                                                  t['f1'] + 1,
+                                                  t['f1'],
                                                   t['f2'],
-                                                  True,
```

Comment:
This makes it so that the BED interval "5 100 105" will get printed as "(5:100-105]" (right?), which I can't imagine is what anybody would expect. I still think the right thing to do is `locus_interval_expr(t['f0'], t['f1']+1, t['f2']+1, True, False)`. Its logically equivalent, `toLocusInterval` handles all the edge cases correctly, and it prints as the default Hail half-closed interval.

I'm confident I understand the BED interval encoding now, and I'm happy to discuss, but I'll approve whatever you think is best. I don't want to hold this PR up any longer.